OpenMEEGMaths/include/BlasLapackImplementations/OpenMEEGMathsOpenBLASConfig.h

```diff
@@ -9,6 +9,8 @@
     #define LAPACK_COMPLEX_CUSTOM
     #define lapack_complex_float _Fcomplex
     #define lapack_complex_double _Dcomplex
+#elif defined(__clang__)
+    #pragma clang diagnostic ignored "-Wc99-extensions"
```

Comment:
And the one `clang` thing we had to ignore because OpenBLAS etc. use `_Complex` from C99